Makefile

```diff
@@ -19,18 +19,15 @@ LIB = libnsdl.a
 SRCS := \
 	source/libNsdl/src/sn_grs.c \
 	source/libNsdl/src/sn_nsdl.c \
-	source/libCoap/src/sn_coap_protocol.c \
-	source/libCoap/src/sn_coap_parser.c \
-	source/libCoap/src/sn_coap_header_check.c \
-	source/libCoap/src/sn_coap_builder.c \
 
 override CFLAGS += -DVERSION='"$(VERSION)"'
 
 override CFLAGS += -Isource/libNsdl/src/include/
-override CFLAGS += -Isource/libCoap/src/include/
 SERVLIB_DIR := ../libService
 override CFLAGS += -I$(SERVLIB_DIR)/libService
 override CFLAGS += -Insdl-c/
+override CFLAGS += -I../libcoap/libcoap/
+override CFLAGS += -I../libcoap/source/include/
```

Comment:
Unfortunately sn_grs has some internal data handling with coap structure , hence it needs these private headers. 
Is it ok, if we remove that dependency as a separate task, once we have this new structure in place ?